screen-search/src/main/java/com/ivy/search/SearchViewModel.kt

```diff
@@ -39,15 +41,18 @@ class SearchViewModel @Inject constructor(
 
     @Composable
     override fun uiState(): SearchState {
+        val prevSearchString = retrieveSearchHistory()
```

Comment:
Shared preferences call on the UI thread in Composable which can be recomposed many times may lead to bad performance. Can we do that in the LaunchedEffect?

Reply:
Anhaa, will do that.